guides/runbooks/secrets.md

```diff
@@ -0,0 +1,411 @@
+# Secrets
+
+This document attempts to cover the 'where' of various secrets, and what they control.
+
+## 1. GitHub Organization Level Secrets
+
+The following secrets are stored as `Actions` secrets in the `pulsar-edit` organization.
+
+Accessible from `https://github.com/organizations/pulsar-edit/settings/secrets/actions`.
+
+### 1.1 `NPM_PUBLISH_TOKEN`
+
+#### Description
+
+This token allows publish access to our `@pulsar-edit` NPM account. Likely this token was generated by a user on the NPM organization that has publish access.
```

Comment:
I should've asked this before this landed, but: how is membership in the NPM organization decided? Do we have a bus factor there as well?